components/compact-call-out/readme.md

```diff
@@ -0,0 +1,21 @@
+Similar to the Call-Out component, this is a more compact section with smaller copy
```

Comment:
**Nit**: Do we want to add a hyphen whenever we mention the `call out` component? In the `call-out/readme.md` file, you didn't add the hyphen when mentioning the component's name.